### ISSUE TITLE
HER2 positive breast tissue

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10088]
**Preferred term label:**
HER2 positive breast tissue

**Synonyms**
human epidermal growth factor receptor 2

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1960398

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Breast ductal carcinoma in situ

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Added as HP:0020324
label: Increased breast-tissue HER2 expression
def:Increased level of human epidermal growth factor receptor 2 (HER2) expression in breast tissue.
comment: HER2 gene amplification assessed by in situ hybridization (ISH) or protein overexpression assessed by IHC remains the primary predictor of responsiveness to HER2-targeted therapies in breast cancer.
PMID:29846122
